cv_next/server/api/cvs.ts

```diff
@@ -17,8 +20,9 @@ export const revalidate = Definitions.CVS_REVALIDATE_TIME_IN_SECONDS;
  * @return {Promise<CvModel | null>} The retrieved CV or null if not found
  */
 export async function getCvById(cvId: string): Promise<CvModel | null> {
+  'use server';
```

Comment:
This is not how `use server` works. If you put `use server` on the entire file, the entire file would be referred as server file.

Reply:
ok thanks Yechiam